<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imperium Behavior Tree Editor[*]</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="Tree" name="tree">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>250</number>
      </attribute>
      <column>
       <property name="text">
        <string>Node</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Comment</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="m_openTreeAction"/>
    <addaction name="m_newTreeAction"/>
    <addaction name="separator"/>
    <addaction name="m_saveAction"/>
    <addaction name="m_saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="m_quitAction"/>
   </widget>
   <addaction name="menuNew"/>
  </widget>
  <action name="m_newTreeAction">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="m_saveAction">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="m_saveAsAction">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="m_quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="m_newNodeAction">
   <property name="text">
    <string>New node</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="m_deleteNodeAction">
   <property name="text">
    <string>Delete node</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="m_openTreeAction">
   <property name="text">
    <string>Open Tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="m_includeSubtreeAction">
   <property name="text">
    <string>Include subtree</string>
   </property>
   <property name="toolTip">
    <string>Include a subtree from another file</string>
   </property>
  </action>
  <action name="m_editNodeAction">
   <property name="text">
    <string>Edit node</string>
   </property>
   <property name="toolTip">
    <string>Edit the current node</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Tree</class>
   <extends>QTreeWidget</extends>
   <header>Tree.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
